pkg/features/features.go

```diff
@@ -90,4 +90,9 @@ const (
 	// owner: @mvladev
 	// alpha: v1.15.0
 	SeedKubeScheduler featuregate.Feature = "SeedKubeScheduler"
+
+	// ReversedVPN moves the openvpn server to the seed.
+	// owner: @scheererj @docktofuture
+	// alpha: v1.20.0
```

Comment:
```suggestion
	// alpha: v1.22.0
```

Reply:
Nice catch.
Fixed with the latest commit.